.openshift-ci/build/Dockerfile.build-bundle

```diff
@@ -1,4 +1,4 @@
-FROM quay.io/stackrox-io/apollo-ci:scanner-test-0.3.45
+FROM quay.io/stackrox-io/apollo-ci:scanner-test-0.3.46
```

Comment:
@rukletsov for future reference, updating these aren't really required since they will be overwritten in CI